app.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'person'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  attr_accessor :books, :people
+
+  def list_books
+    @books.each_with_index do |book, index|
+      puts "#{index}) Title: #{book.title} Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each_with_index do |person, index|
+      puts "#{index}) Name: #{person.name} ID: #{person.id}"
+    end
+  end
+
+  def create_person(age, name: 'Unknown')
+    person = Person.new(age, name: name)
+    @people.push(person)
+  end
```

Comment:
- [ ] When I try to create a person, the app doesn't ask me if I want to create a student or teacher. It directly asks for input. Please make sure when the user selects the option to create a person, The app should ask if the user wants to create a student or teacher and request appropriate input. It's the project requirement that the app should work the same way after refactoring.